cmd/avalanche-network-runner/control/control.go

```diff
@@ -341,6 +344,116 @@ func restartNodeFunc(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func newAttachPeerCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "attach-peer [options]",
+		Short: "Attaches a peer to the node.",
+		RunE:  attachPeerFunc,
+	}
+	cmd.PersistentFlags().StringVar(
+		&nodeName,
+		"node-name",
+		"",
+		"node name to attach a peer to",
+	)
+	return cmd
+}
+
+func attachPeerFunc(cmd *cobra.Command, args []string) error {
+	cli, err := client.New(client.Config{
+		LogLevel:    logLevel,
+		Endpoint:    endpoint,
+		DialTimeout: dialTimeout,
+	})
+	if err != nil {
+		return err
+	}
+	defer cli.Close()
+
+	ctx, cancel := context.WithTimeout(context.Background(), requestTimeout)
+	resp, err := cli.AttachPeer(ctx, nodeName)
+	cancel()
+	if err != nil {
+		return err
+	}
+
+	color.Outf("{{green}}attach peer response:{{/}} %+v\n", resp)
+	return nil
+}
+
+var (
+	peerID             string
+	msgOp              uint32
+	msgBytesB64        string
+	msgBytesThrottling bool
```

Comment:
Should we include this? I don't think there will ever be a time when we want the test peer to throttle messages (i.e. ` req.BytesThrottling` should always be `true` in calls to `NewTestMsg` so that we bypass throttling.)